Userland/Utilities/strace.cpp

```diff
@@ -36,24 +137,328 @@ static void handle_sigint(int)
     }
 }
 
-static String format_syscall(syscall_arg_t syscall_index, syscall_arg_t arg1, syscall_arg_t arg2, syscall_arg_t arg3, syscall_arg_t res)
+static void copy_from_process(const void* source_p, void* target_p, size_t size)
+{
+    const char* source = static_cast<const char*>(source_p);
```

Comment:
No need to state the type twice.
```suggestion
    auto source = static_cast<const char*>(source_p);
```

Reply:
(same below for `target`)

Reply:
Thanks for pointing out, it was a oversight, fixed now.